<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generowanie labiryntu</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="gameWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_minus">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_plus">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcd_time">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="welcomeWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_new">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Nowa gra</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_game">
         <property name="text">
          <string>SZYBKA GRA:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_easy">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Łatwa</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_medium">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Średnia</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_hard">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Trudna</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_expert">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Ekspert</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_leader">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Najlepsze czasy...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_load">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Wczytaj</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_help">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Pomoc</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_exit">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Wyjście</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignBottom">
     <widget class="QWidget" name="infoWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>260</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_size">
         <property name="text">
          <string>size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_difficulty">
         <property name="text">
          <string>difficulty</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>259</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>278</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Szybka gra</string>
    </property>
    <addaction name="action_easy"/>
    <addaction name="action_medium"/>
    <addaction name="action_hard"/>
    <addaction name="action_expert"/>
    <addaction name="separator"/>
    <addaction name="action_leader"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Labirynt</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_blackout"/>
    <addaction name="action_load"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
    <addaction name="action_help"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGame"/>
  </widget>
  <action name="action_load">
   <property name="text">
    <string>Wczytaj zapisany...</string>
   </property>
   <property name="toolTip">
    <string>Otwórz z pliku...</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Zapisz</string>
   </property>
   <property name="toolTip">
    <string>Zapisz</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="text">
    <string>Zapisz jako...</string>
   </property>
   <property name="toolTip">
    <string>Zapisz jako...</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Wyjdź z gry</string>
   </property>
  </action>
  <action name="action_easy">
   <property name="text">
    <string>Łatwy</string>
   </property>
  </action>
  <action name="action_medium">
   <property name="text">
    <string>Średni</string>
   </property>
  </action>
  <action name="action_hard">
   <property name="text">
    <string>Trudny</string>
   </property>
  </action>
  <action name="action_expert">
   <property name="text">
    <string>Ekspert (~15s)</string>
   </property>
  </action>
  <action name="action_leader">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rankingi</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>Nowa gra</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>Zamknij labirynt</string>
   </property>
  </action>
  <action name="action_blackout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ograniczony widok</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Pomoc</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
